app/models/standard_loan_data_export.rb

```diff
@@ -0,0 +1,94 @@
+class StandardLoanDataExport < DataExport
+
+  HEADERS = [
+    'loan_id',
+    'name',
+    'division',
+    'cooperative',
+    'country',
+    'postal_code',
+    'status',
+    'actual_end_date',
+    'actual_first_payment_date',
+    'actual_first_interest_payment_date',
+    'projected_end_date',
+    'projected_first_payment_date',
+    'projected_first_interest_payment_date',
+    'signing_date',
+    'loan_type',
+    'currency',
+    'amount',
+    'primary_agent',
+    'secondary_agent',
+    'sum_of_disbursements',
+    'sum_of_repayments',
+    'change_in_principal',
+    'change_in_interest'
+  ]
+
+  # Subclass exists only to implement process_data. No additional public methods should be added to this subclass.
+  def process_data
+    @child_errors = []
+    data = []
+    data << header_row
+    Loan.find_each do |l|
+      begin
+        data << hash_to_row(loan_data_as_hash(l))
+      rescue => e
+        @child_errors << {loan_id: l.id, message: e.message}
```

Comment:
This does the same format as the errors in the loan update task. There isn't any enforcement of this format except that the task/show view assumes it.